Fixes #7089 - Updates about_Parameter_Sets (#7101)
MicrosoftDocs/PowerShell-Docs: reference/5.1/Microsoft.PowerShell.Core/About/about_Parameter_Sets.md

The following requirements apply to all parameter sets.

> - Each parameter set must have at least one unique parameter. Meaning, each
>   parameter set must be a unique combination of parameters even if only one
>   parameter is different than the other sets. If possible, make this parameter
>   a mandatory parameter.

Reviewer: ```suggestion
- Each parameter set must have a unique combination of parameters. If possible,
   at least one of the unique parameters should be a mandatory parameter.
```
